### PR TITLE
Update enablemfa.md

### PR DESCRIPTION
**Pull request by AhsenBaig**
Updated instructions for SMS as Multifactor isn't an option anymore.

## Description:
- **What's changed?** 
Updated documentation for setting up SMS.

### CODE REVIEW

**Comment by AhsenBaig** [comment]
Document should match https://help.okta.com/oie/en-us/Content/Topics/identity-engine/authenticators/configure-phone.htm

```
1. In the Admin Console, go to Security > Authenticators.
2. On the Setup tab, click Add Authenticator.
3. Click Add on the Phone tile.
4. Choose whether verification messages are sent via Voice call, SMS, or select both options.
```

File: packages/@okta/vuepress-site/docs/guides/mfa/main/sms/enablemfa.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-1. From the Admin Console, select **Security** from the main menu and then **Multifactor**.
-2. On the **Factor Types** tab, select **SMS Authentication** from the left navigation.
-3. Click the **Inactive** drop-down box, and then select **Activate**.
+1. From the Admin Console, select **Security** from the main menu and then **Authenticators**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/main/sms/enablemfa.md:1]:
```suggestion
1. In the Admin Console, go to **Security** > **Authenticators**.
```

File: packages/@okta/vuepress-site/docs/guides/mfa/main/sms/enablemfa.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
+2. Click the **Add Authenticator** button, select **Phone** for SMS/Voice.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/main/sms/enablemfa.md:2]:
```suggestion
2. Click **Add Authenticator**, and then click **Add** on the Phone tile.
```

File: packages/@okta/vuepress-site/docs/guides/mfa/main/sms/enablemfa.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
...
+3. Click the Add button associated with the service.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/main/sms/enablemfa.md:3]:
```suggestion
3. Choose whether verification messages are sent through **Voice call**, **SMS**, or select both options.
4. Select when users can use the Phone authenticator:
    * **Authentication and password reset**
    * **Authentication (MFA/SSO)
    * **Password reset**
5. Click **Add**.
```

**Review by susanharper-okta — COMMENTED** [review]
couple edits

File: packages/@okta/vuepress-site/docs/guides/mfa/main/sms/enablemfa.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
-1. From the Admin Console, select **Security** from the main menu and then **Multifactor**.
-2. On the **Factor Types** tab, select **SMS Authentication** from the left navigation.
-3. Click the **Inactive** drop-down box, and then select **Activate**.
+1. In the Admin Console, go to **Security** > **Authenticators**.
+2. Click **Add Authenticator**, and then click **Add** on the Phone tile.
+3. Choose whether verification messages are sent through **Voice call**, **SMS**, or select both options.
+4. Select when users can use the Phone authenticator:
+    * **Authentication and password reset**
+    * **Authentication (MFA/SSO)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/mfa/main/sms/enablemfa.md:6]:
```suggestion
    * **Authentication** (MFA/SSO)
```

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**